backend/webgpu/operators/unary.ts

```diff
@@ -29,6 +29,7 @@ export const abs = unary("return abs(a);");
 export const linear = unary("return a;");
 export const neg = unary("return -a;");
 export const relu = unary<"f32" | "i32">((type) => `return max(a, 0${type});`);
+export const relu6 = unary<"f32" | "i32">((type) => `return clamp(a, 0${type}, 6.0);`);
```

Comment:
The `6.0` should be changed to `6${type}` to match the input type in case it is `i32` and not `f32` but i doubt this would ever be used with `i32` as input type...